### ISSUE TITLE
Villagers can open locked doors

### ISSUE THREAD

**Issue by imanairconditioner** [Melonslise/Locks#49]
The expected behaviour is that Villagers just rattle the lock and walk away.

**Comment by MercurialPony** [comment]
Not really a bug, but definitely something I should consider adding. Might require a coremod hack though, not sure

**Comment by imanairconditioner** [comment]
> Not really a bug,

?

> but definitely something I should consider adding.

Good!

**Comment by MercurialPony** [comment]
> ?

Well, yes, I never actually specifically wrote the code to stop villagers or other entities except players from using locked blocks

**Comment by auggieciorra** [comment]
Whens this feature gonna be implemented for 1.12

**Comment by MythrilBagels** [comment]
> but definitely something I should consider adding.

Yes please, just had a villager walk into my house and steal my bed. It would be kinda cool if Villagers could somehow hold a key of their own and were able to pathfind through specifically their door, although I feel like that would be quite difficult
